content/zh/docs/reference/issues-security/issues.md

```diff
@@ -1,16 +1,33 @@
 ---
 title: Kubernetes 问题追踪
 weight: 10
+aliases: [/cve/,/cves/]
```

Comment:
@tanjunchen This PR reintroduced the bug in #18691. 

Aliases in localized versions need the leading slash removed. These should instead be:
```
aliases: [cve/, cves/]
```

Reply:
ok